ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -75,7 +126,9 @@ private[platform] trait LedgerReadDao extends ReportsHealth {
       endInclusive: Offset,
   )(implicit loggingContext: LoggingContext): Source[(Offset, ConfigurationEntry), NotUsed]
 
-  def transactionsReader: TransactionsReader
+  def transactionsReader: LedgerDaoTransactionsReader
```

Comment:
we could also mix in the traits, but ATM this is good since it will result in low impact change